Source/Diagnostics/Diagnostics.cpp

```diff
@@ -243,7 +243,114 @@ Diagnostics::InitBaseData ()
         m_flush_format = std::make_unique<FlushFormatCheckpoint>() ;
     } else if (m_format == "ascent"){
         m_flush_format = std::make_unique<FlushFormatAscent>();
-    } else if (m_format == "sensei"){
+        std::cout<< " m_diag_name " <<m_diag_name<<std::endl;
+        amrex::ParmParse pp_diag_viz(m_diag_name);
+        int write_yaml_file = 0;
+        pp_diag_viz.query("viz", write_yaml_file);
+        if (write_yaml_file){
```

Comment:
That way we only write from one rank:
```suggestion
        if (write_yaml_file && ParallelDescriptor::IOProcessor()){
```

Reply:
Done

Reply:
@lge0303 did you push the changes you mentioned to the PR's branch? I still see the old version on GitHub :)